js/sign.js

```diff
@@ -2,100 +2,200 @@ const useremail = document.querySelector('#useremail')
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        "email": useremail.value,
+        "password": password.value,
+      }),
+    })
+    if (response.status === 200){
+      if (useremail.value === 'test@codeit.com' && password.value === 'sprint101'){
+        location.href = "/folder";
+        return;
+      }
     }
+  }catch{
+    checkLogin();
+  };
+
+  try {
+    const response = await fetch("https://bootcamp-api.codeit.kr//api/sign-up", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        "email": useremail.value,
+        "password": password.value,
+      }),
+    })
+    if (response.status === 200){
+      if (useremail.value && (password.value || password2.value)){
+        if (signInputEmail.lastChild.tagName !== "P" && (signInputPassword.lastChild.tagName !== "P" || signInputPassword2.lastChild !== "P")){
+          location.href = "/folder";
+          return; 
+        }
+      }
+    }
+  } catch {
+    checkLogin();
   }
-  if (useremail.value === 'test@codeit.com' && password.value === 'codeit101'){
-    location.href = "/folder";
-    return;
-  }
+}
+  
+
+function enterkey(e) {
+	if (e.keyCode == 13) {
+    console.log(e.keyCode);
+    submitForm(e);
+    }
+}
+
+function checkLogin(){
+  if (signInputPassword.lastChild.tagName === "P"){
+    signInputPassword.lastChild.remove();
+  } 
+  if (useremail.nextSibling){
+    useremail.nextSibling.remove();
+  } 
+  paintErr('이메일을 확인해주세요', signInputEmail);
+  paintErr('비밀번호를 확인해주세요', signInputPassword);
 }
 
+const toggleEye = document.querySelectorAll('.eye');
+function toggleEyeHandler(){
+  for (let i = 0; i < toggleEye.length; i++){
+    if (toggleEye[i].classList.contains('eye-off')){
+      toggleEye[i].classList.remove('eye-off');
+      toggleEye[i].classList.add('eye-on');
+      toggleEye[i].childNodes[1].setAttribute('src', 'imgs/sign/eye-on.png');
+      password.type = 'text';
+      password2.type = 'text';
+    } else if (toggleEye[i].classList.contains('eye-on')){
+      toggleEye[i].classList.remove('eye-on');
+      toggleEye[i].classList.add('eye-off');
+      toggleEye[i].childNodes[1].setAttribute('src', 'imgs/sign/eye-off.svg');
+      password.type = 'password';
+      password2.type = 'password';
+    }
+  }
+}
+toggleEye.forEach(eye => {
+  eye.addEventListener('click', toggleEyeHandler);
+});
 
 
-useremail.addEventListener('focusout', chackingEmail);
-password.addEventListener('focusout', chackingPassword);
+useremail.addEventListener('focusout', checkEmail);
+password.addEventListener('focusout', checkPassword);
 
 signForm.addEventListener('submit', submitForm);
-password2.addEventListener('focusout', doubleCheakingPassword);
+signForm.addEventListener('keyup', enterkey);
+
+password2.addEventListener('focusout', doubleCheckPassword);
```

Comment:
```suggestion
password2?.addEventListener('focusout', doubleCheckPassword);
```
로그인 페이지에서는 password2가 없어서 에러가 나고 있어요. 

<img width="597" alt="image" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/60066472/434e0cb4-2930-4572-92e3-52836296dbc5">